include/gradualizer.hrl

```diff
@@ -10,26 +10,28 @@
 %%
 %%     N = ?annotate_type( Message, non_neg_integer() )
 %%
-%% The macro ?assert_type/2 can be used to override any type detected by
-%% Gradualizer.  For example, the programmer may know that the length of
-%% a list is within a certain range, rather than any non_neg_integer():
+%% The macro ?assert_type/2 can be used to refine (downcast) a type propagated
+%% by Gradualizer.  For example, the programmer may know that the length of a
+%% list is within a certain range, rather than any non_neg_integer():
 %%
 %%     Arity = ?assert_type( length(Args), arity() )
 %%
-%% The function '::' can also be used directly if the type is quoted:
+%% The functions '::'/2 and ':::'/2 can also be used directly if the type is
+%% quoted:
 %%
 %%     N = '::'(Message, "non_neg_integer()")
 %%
-%% Gradualizer occurences of the '::'/2 function and adjusts type checking
-%% accordingly.  The macros are supplied only for convenience.
-
--compile({inline, ['::'/2]}).
+%% Gradualizer detects occurences of the functions '::'/2 and ':::'/2 and
+%% adjusts type checking accordingly.  The macros are supplied only for
+%% convenience.
+%%
+-compile({inline, ['::'/2, ':::'/2]}).
```

Comment:
It's really helpful that you've written this documentation. I think it would be even more helpful if you further clarified the distinction between `::` and `:::`. Right now it's not clear which one does annotation and which does assertion.

Reply:
Oh, that's true.